### ISSUE TITLE
Incremental Brute Force 

### ISSUE THREAD

**Issue by Solpadein911** [ryancdotorg/brainflayer#153]
Please tell me why on the command samples **(./brainflayer -v -I **000000000000000000000000000000000000000000000000000000000001** -b btcaddress.blf -o found.txt)**  the number one is put at the end or beginning. As far as I know, the key is a closed format may not contain a digit one at the beginning or end.
**Hex: b49c0a968de2df1367dfd9301a26b9fdbc5db0bfe892f453f2e0156873f8ef74** 
Or am I mistaken in something?

**Comment by ryancdotorg** [comment]
I'm not sure I understand the question. The key is an integer scalar on the interval [1,0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364140], represented as 64 hexadecimal digits. The example listed is 000...001 because all zeros isn't a valid key.

**Comment by Solpadein911** [comment]
Thank you for your responsiveness. Your answer raised a few more questions.
**The example listed is 000...001 because all zeros isn't a valid key.**
I checked the program can run without the number one, only 64 zeros. Have the command in this form **. / brainslayer-you 00000000000000000000000000000000000000000000000001-b btcaddress. tvo-o found.txt** is the number 0 a random character?

**Comment by Solpadein911** [comment]
Thank you for your responsiveness. Your answer raised a few more questions.
 **The example listed is 000...001 because all zeros isn't a valid key.**
I checked the program can run without the number 1, only 64 zeros. running the command in this form. / brainslayer-you 00000000000000000000000000000000000000000000000001-b btcaddress. tvo-o found.txt the number 0 will it be a random character?

**Comment by ryancdotorg** [comment]
The number zero is zero. An all zero key will "work" in brainflayer, but the results will be buggy. No attempt to detect this is made for performance reasons.

**Comment by Solpadein911** [comment]
Last question: In your memory, have there been cases when the Incremental brute force mode gave someone a positive result??
 I started with it, but it seems that this method of brute force is infinite and unlikely.
**You have a lot of practice, and I want to ask what would you recommend the method of brute force?** Thanks

**Comment by ryancdotorg** [comment]
The brute force mode is only useful for looking at "interesting" ranges of keys. It's astronomically unlikely to ever find a match otherwise.

**Comment by Solpadein911** [comment]
You have a lot of practice, and I want to ask, what method of working with the program do you recommend?

**Comment by ryancdotorg** [comment]
At the risk of being blunt, even if you have a system of ethics that deems taking any coins you have the keys for as acceptable, there are better ways to make money. Pretty much everything that could be cleaned out already has been by people who wrote their own GPU-based tools for it. If you have free electricity, there are better ways to monetize it, and if you pay for power you'll be better off buying lottery tickets.

**Comment by Solpadein911** [comment]
Yes, everything is exactly as you say.
I've already guessed it myself. Smart guys like you have already come up from different angles and done a good job of it. But still, I'm interested in playing and getting this experience. Thank you for support. 
Question: 
I tested the program and added the passphrase < < password>>**./brainflayer -v -b btcaddress.blf -i wordlist.txt -o found.txt  with** this command, I find a match. But if I try to do it through a Crunch:
 **crunch 8 8 password  ./brainflayer -v -c cu -t priv -x -b btcaddress.blf -o found.txt**
 then there are no matches,I tried it repeatedly :(
